### PR TITLE
add install script

### PR DESCRIPTION
**Pull request by justin1121**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	rm "$exe.tar.gz"
+
+	echo "nitrogen was installed successfully to $exe"
+	if command -v nitrogen >/dev/null; then
+		echo "Run 'nitrogen --help' to get started"
+	else
+		case $SHELL in
+		/bin/zsh) shell_profile=".zshrc" ;;
+		*) shell_profile=".bashrc" ;;
+		esac
```
Comment (ChrisFriesen) [inline install.sh:71]: you're not actually using shell_profile, can remove the case statement

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+		*)
+			echo "Error: Official nitrogen builds for Linux arm64 are not available" 1>&2
+			exit 1
+			;;
+		esac
+	fi
+
+	# https://github.com/capeprivacy/nitrogen/releases/download/v0.1.0/nitrogen_v0.1.0_x86_64-apple-darwin.zip
+	# https://github.com/capeprivacy/nitrogen/releases/download/v0.1.0/nitrogen_v0.1.0_x86_64-unknown-linux-musl.tar.gz
+	# https://github.com/capeprivacy/nitrogen/releases/download/v0.1.0/nitrogen_v0.1.0_x86_64-pc-windows-gnu.zip
```
Comment (ChrisFriesen) [inline install.sh:40]: are these here as an example?

File: install.sh (hunk 1 above)
Comment (justin1121) [reply install.sh:71]: nice catch!

File: install.sh (hunk 2 above)
Comment (justin1121) [reply install.sh:40]: whoops meant to remove these

**Review by ChrisFriesen — APPROVED** [review]

**Pull request merged**